bot/exts/core/error_handler.py

```diff
@@ -58,7 +58,10 @@ async def on_command_error(self, ctx: Context, error: CommandError) -> None:
             embed = self._get_error_embed("Command not found", str(error))
         else:
             # If we haven't handled it by this point, it is considered an unexpected/handled error.
-            log.exception(f"Error executing command invoked by {ctx.message.author}: {ctx.message.content}")
+            log.exception(
+                f"Error executing command invoked by {ctx.message.author}: {ctx.message.content}",
+                exc_info=error,
```

Comment:
I don't understand why this is needed, `log.exception` automatically pulls `exc_info` from `sys.exec_info()` if not passed explitily. Is `sys.exec_info()` not available from listeners like this?

Reply:
No, it does not seem to be. My theory is that it's not done within a `except`, so `sys.exc_info()` doesn't know about it. See [the Sentry issue](https://python-discord.sentry.io/issues/5126046667/?project=4506666249945088&query=&referrer=issue-stream&statsPeriod=90d&stream_index=5,).

Reply:
how annoying